### PR TITLE
"Mini Airport" Map by DiamondPlane

### PR DESCRIPTION
**Pull request by AnyBlue**
New map "Mini Airport"

### CODE REVIEW

**Comment by HexyBubbled** [comment]
Gonna test :+1:

**Comment by Panquesito7** [comment]
`map.mts` and `map.conf` files are available.

File: mini_airport/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+team.2.color = blue
+team.2.pos = (-24,-24,-100)
+team.1.pos = (6,-38,100)
+team.1.color = red
+team.1 = red
+author = DiamondPlane
+rotation = z
+h = 140
+team.2 = blue
+name = Mini Airport
```
Comment (Panquesito7) [inline mini_airport/map.conf:10]: Maybe add a hint?

Comment (AnyBlue) [reply mini_airport/map.conf:10]: What a hint? There no diamonds/mese

**Pull request closed without merging**